src/App.jsx

```diff
@@ -53,7 +53,10 @@ export function App() {
 				<Route path="/" element={<Layout />}>
 					<Route index element={<Home />} />
 					<Route path="/list" element={<List data={data} />} />
-					<Route path="/add-item" element={<AddItem />} />
+					<Route
+						path="/add-item"
+						element={<AddItem addItem={addItem} listToken={listToken} />}
```

Comment:
My only suggestion here is that instead of importing `addItem` and then passing it as a prop to the `AddItem` component, you can import directly in the `AddItem` component so you have fewer props to pass down.

Reply:
@luisaugusto I've made this change that you've suggested.